src/app/pages/observations/observation-list.component.html

```diff
@@ -1,14 +1,36 @@
-<otp-tabs [tabs]="tabs"></otp-tabs>
 <div class="content">
   <div>
-    <a [routerLink]="[private/observations/new]">New observation</a>
+    <a href="private/observations/new">New observation</a>
```

Comment:
This was causing a weird issue resulting in the application to go to a different route...